Ghost protocol upgrade files/memory-api-service.ts

```
@@ -0,0 +1,374 @@
+      console.log(`📖 Docs: http://localhost:${port}/docs`);
+    });
+  }
+
+  public getApp(): express.Application {
+    return this.app;
+  }
+}
+
+// memory-service/src/controllers/MemoryController.ts
+import { Request, Response } from 'express';
+import { MemoryCreateRequest, MemorySearchRequest, APIResponse } from '../types/api';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'APIResponse'. Consider to delete it or rename it.

'APIResponse' is defined here:


(lint/suspicious/noRedeclare)